src/index.tsx

```diff
@@ -40,12 +40,26 @@ export type SnapyrConfigOptions = {
   snapyrEnvironment: SnapyrEnvironment;
 };
 
+export enum SnapyrInAppActionType {
+  Custom = 'custom',
+  Overlay = 'overlay',
+}
+
+export enum SnapyrInAppPayloadType {
+  JSON = 'json',
+  HTML = 'html',
+}
+
+export declare type SnapyrInAppContent =
+  | { payloadType: SnapyrInAppPayloadType.JSON; payload: Record<string, any> }
```

Comment:
The leading pipe here is confusing to me. Is that a formatting thing or actual syntax?

Reply:
Thanks @jonaldinger ! Just a formatting thing that was auto applied by the linter. This seems to be a fairly standard linting rule (we have it in the CP repo as well). I agree it's weird but I guess it's similar to including a trailing comma on the last line of a map / list - lets you reorder the lines later, if you want, without changing characters.

Reply:
Cool, TIL. Thanks!